src/Resources/config/core.xml

```diff
@@ -32,7 +32,7 @@
             <argument type="service" id="sonata.block.context_manager"/>
             <argument type="service" id="event_dispatcher"/>
             <argument type="service" id="sonata.cache.manager" on-invalid="ignore"/>
-            <argument type="service" id="sonata.block.cache.handler.default" on-invalid="ignore"/>
+            <argument type="service" id="sonata.block.cache.handler" on-invalid="ignore"/>
```

Comment:
this was a bug. It never used the configured handler service alias... :see_no_evil: